vulnerabilities/api.py

```diff
@@ -141,3 +166,22 @@ class VulnerabilityViewSet(viewsets.ReadOnlyModelViewSet):
     paginate_by = 50
     filter_backends = (filters.DjangoFilterBackend,)
     filterset_class = VulnerabilityFilterSet
+
+    # `fetch` is a placeholder
+    # TODO: Find a good name for this endpoint
+    @action(detail=False, methods=["post"])
+    def fetch(self, request):
+        filter_list = Q()
+        response = {}
+        # TODO: Do some validation here of request body
+        for cve_id in request.data["vulnerabilities"]:
+            filter_list |= Q(cve_id=cve_id)
```

Comment:
You could use the `__in` lookup instead.

Reply:
Right :)